Tools/environment_install/install-prereqs-mac.sh

```diff
@@ -1,47 +1,93 @@
 #!/bin/bash
-echo "Checking homebrew..."
+
+echo "`tput setaf 2`Checking homebrew...`tput sgr0`"
```

Comment:
Can we create a "heading" function rather than putting all of these `tput`'s inline, please?